{"name": "test_api_query[10000]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <test.TestAPICase object at 0x7f8848fc4c50>, id = 10000\n\n    @allure.story(\"query user\")\n    @pytest.mark.parametrize('id', \\\n                             [conf['api_test']['query_api']['param']['id'], \\\n                              conf['api_test']['query_api']['param']['failed_id']])\n    def test_api_query(self, id):\n        test_conf = conf['api_test']['query_api']\n        url = test_conf['host'] + test_conf['path']\n        payload = {}\n        headers = conf['api_test']['headers']\n    \n        response = requests.request(\"GET\", url, headers=headers, data=payload)\n        print(response.text)\n        assert response.status_code == 200\n        found = 0\n        for usr in json.loads(response.text):\n            for key in usr:\n                if usr[key] == id:\n                    found = 1\n                    break\n>       assert found  # 返回的查询列表中，有正确的用户id\nE       assert 0\n\ntestcase/test.py:77: AssertionError"}, "parameters": [{"name": "id", "value": "10000"}], "start": 1706532886157, "stop": 1706532887175, "uuid": "cb8e6718-c1a6-4b2f-a99c-337e6a9e5d38", "historyId": "15af48bcb2a9bafcc5a4d5b12fb36fb9", "testCaseId": "382f41298ea2b9d0dcdc5f82a96ffddb", "fullName": "testcase.test.TestAPICase#test_api_query", "labels": [{"name": "story", "value": "query user"}, {"name": "feature", "value": "API testcase"}, {"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test"}, {"name": "subSuite", "value": "TestAPICase"}, {"name": "host", "value": "joannadeMacBook-Air.local"}, {"name": "thread", "value": "28018-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test"}]}